commands/member.js

```diff
@@ -13,16 +13,17 @@ exports.role = function(message, args){
         message.channel.send(`**These are game roles you're allowed to join:** \n${global.util.listToString(global.GAME_ROLES)} \nUse \`!role <role_name>\` to join a role \nUse \`!rmrole <role_name>\` to leave a role`)
         return;
     }
-
+    
     let role = message.guild.roles.find(`name`, args.join(' '));
+    logger.info(`User ${message.author.username} joining role ${role}`);

```

Comment:
Maybe change to 'attempting to join role' since this is before the role exist check